application/jaxrs-hazelcast-quarkus/src/test/java/webapp/tier/service/HazelcastMqServiceTest.java

```diff
@@ -22,6 +28,7 @@
 	HazelcastMqService svc;
 
 	private static HazelcastInstance mockInstance;
+	String respbody = "message: Hello k8s-3tier-webapp with quarkus";
```

Comment:
Codacy found an issue: [Use explicit scoping instead of the default package private level](https://app.codacy.com/manual/yurake/k8s-3tier-webapp/pullRequest?prid=5923183)